mayastor-test/e2e/node_disconnect/node_disconnect_iscsi_reject/node_disconnect_iscsi_reject_test.go

```diff
@@ -0,0 +1,56 @@
+package node_disconnect_iscsi_reject_test
+
+import (
+	"e2e-basic/common"
+	disconnect_lib "e2e-basic/node_disconnect/lib"
+	"fmt"
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+var (
+	g_nodeToKill       = ""
+	g_otherNodes       []string
+	g_uuid             = ""
+	g_disconnectMethod = "REJECT"
+)
+
+func lossTest() {
+	g_nodeToKill, g_otherNodes = disconnect_lib.GetNodes(g_uuid)
+	disconnect_lib.LossTest(g_nodeToKill, g_otherNodes, g_disconnectMethod, g_uuid)
+}
+
+func TestNodeLoss(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Node Loss iSCSI reject")
+}
+
+var _ = Describe("Mayastor node loss test", func() {
+	It("should verify behaviour when a node becomes inaccessible", func() {
+		lossTest()
+	})
+})
+
+var _ = BeforeSuite(func(done Done) {
+	logf.SetLogger(zap.LoggerTo(GinkgoWriter, true))
+	common.SetupTestEnv()
+	g_uuid = disconnect_lib.Setup("loss-test-pvc-iscsi", "mayastor-iscsi", "../deploy/fio_iscsi.yaml")
+	close(done)
+}, 60)
+
+var _ = AfterSuite(func() {
+	// NB This only tears down the local structures for talking to the cluster,
+	// not the kubernetes cluster itself.
```

Comment:
I've seen these comments copy and pasted in nearly all of our tests. I wonder if there is someway we can make this obvious so that we don't need to keep putting it in.

Reply:
I'll think about that for the next iteration.